diff --git a/build.gradle b/build.gradle
index 7ce1e5bc..562c1ab2 100644
--- a/build.gradle
+++ b/build.gradle
@@ -13,7 +13,7 @@ buildscript {
         classpath 'com.palantir.gradle.failure-reports:gradle-failure-reports:1.10.0'
         classpath 'com.palantir.gradle.gitversion:gradle-git-version:3.1.0'
         classpath 'com.palantir.javaformat:gradle-palantir-java-format:2.40.0'
-        classpath 'com.palantir.gradle.revapi:gradle-revapi:1.7.0'
+        // classpath 'com.palantir.gradle.revapi:gradle-revapi:1.7.0'
         classpath 'com.palantir.javaformat:gradle-palantir-java-format:2.40.0'
         classpath 'gradle.plugin.org.inferred:gradle-processors:3.7.0'
     }
diff --git a/gradle-palantir-java-format/build.gradle b/gradle-palantir-java-format/build.gradle
index 1fbea343..5db1b14a 100644
--- a/gradle-palantir-java-format/build.gradle
+++ b/gradle-palantir-java-format/build.gradle
@@ -2,7 +2,7 @@ apply plugin: 'java-gradle-plugin'
 apply plugin: 'groovy'
 apply plugin: 'com.palantir.external-publish-jar'
 apply plugin: 'com.palantir.external-publish-gradle-plugin'
-apply plugin: 'com.palantir.revapi'
+// apply plugin: 'com.palantir.revapi'
 
 configurations {
     pluginClasspath {
diff --git a/palantir-java-format-jdk-bootstrap/build.gradle b/palantir-java-format-jdk-bootstrap/build.gradle
index 9348a282..4ea782d9 100644
--- a/palantir-java-format-jdk-bootstrap/build.gradle
+++ b/palantir-java-format-jdk-bootstrap/build.gradle
@@ -1,6 +1,6 @@
 apply plugin: 'java-library'
 apply plugin: 'com.palantir.external-publish-jar'
-apply plugin: 'com.palantir.revapi'
+// apply plugin: 'com.palantir.revapi'
 
 dependencies {
     annotationProcessor "org.immutables:value"
diff --git a/palantir-java-format-spi/build.gradle b/palantir-java-format-spi/build.gradle
index 1be41ab3..768049fc 100644
--- a/palantir-java-format-spi/build.gradle
+++ b/palantir-java-format-spi/build.gradle
@@ -1,6 +1,6 @@
 apply plugin: 'java-library'
 apply plugin: 'com.palantir.external-publish-jar'
-apply plugin: 'com.palantir.revapi'
+// apply plugin: 'com.palantir.revapi'
 
 dependencies {
     api 'com.google.guava:guava'
diff --git a/settings.gradle b/settings.gradle
index 2ceb639d..fb5baa83 100644
--- a/settings.gradle
+++ b/settings.gradle
@@ -1,8 +1,8 @@
 rootProject.name = 'palantir-java-format-parent'
 
 include ':gradle-palantir-java-format'
-include ':eclipse_plugin'
-include ':idea-plugin'
+// include ':eclipse_plugin'
+// include ':idea-plugin'
 include ':palantir-java-format-spi'
 include ':palantir-java-format'
 include ':palantir-java-format-jdk-bootstrap'
diff --git a/versions.props b/versions.props
index 714fa90b..665dd744 100644
--- a/versions.props
+++ b/versions.props
@@ -7,7 +7,6 @@ com.google.guava:guava = 33.0.0-jre
 com.google.guava:guava-testlib = 27.0.1-jre
 com.google.testing.compile:compile-testing = 0.21.0
 com.google.truth:truth = 1.4.1
-com.palantir.sls.versions:sls-versions = 1.5.0
 com.netflix.nebula:nebula-test = 10.0.0
 junit:junit = 4.13.2
 org.assertj:assertj-core = 3.25.3
